hub/powertoys/mouse-without-borders.md

```diff
@@ -34,7 +34,7 @@ With the latest version of PowerToys installed, you will see Mouse Without Borde
 
 4. Once the computers are connected, you will be able to switch between them by moving your mouse cursor beyond the edge of the screen, transitioning between computers.
 
-    ![Screenshot of Mouse Without Borders settings on the first computer after connecting.](../images/powertoys-mouse-without-borders-after-connect-1.png)
+    <!-- ![Screenshot of Mouse Without Borders settings on the first computer after connecting.](../images/powertoys-mouse-without-borders-after-connect-1.png) -->
```

Comment:
Is this screenshot goign to be restored later? If not, I would just remove this instead of commenting it out. Same with the one that's commented out on the fancyzones page.

Reply:
Nah, I'm taking it out.